MCLauncher/MainWindow.xaml.cs

```diff
@@ -230,25 +230,25 @@ public partial class MainWindow : Window, ICommonVersionCommands {
 
         private void InvokeRemove(Version v) {
             if (!v.GetStatusBeingRemoved()) {
-                foreach (Process Proc in Process.GetProcesses()) {
-                    if (Proc.ProcessName.Contains("Minecraft")) {
-                        Debug.WriteLine("Killing Minecraft process");
-                        Proc.Kill();
-                    }
-                }
-                Thread.Sleep(1000);
-
                 v.UpdateBeingRemoveStatus(true);
                 foreach (var pkg in new PackageManager().FindPackages(MINECRAFT_PACKAGE_FAMILY)) {
                     if (pkg.InstalledLocation.Path.Contains(v.GameDirectory)) {
+                        foreach (Process Proc in Process.GetProcesses()) {
+                            if (Proc.ProcessName.Contains("Minecraft")) {
```

Comment:
what happens if MCJE is running?

Reply:
Java edition's process name is generally either "java" or "javaw". Not sure about the launcher though.

Reply:
Java edition's process name is generally either "java" or "javaw". Not sure about the launcher though.

Reply:
killing random processes that happen to include "Minecraft" in the name by pure chance is probably a bad idea

Reply:
I know this is a necropost but why not just kill Minecraft.Windows.exe?